rakefile.rb

```diff
@@ -95,6 +101,9 @@
 desc "Execute default tasks"
 task :default => [ :vars, :gitlink, :unit, :integ, :spec, :pack ]
 
+desc "Execute default tasks for .NET Core solution"
+task :dotnet => [ :default, :unit_dotnet, :pack_dotnet]

```

Comment:
This will also execute non-dotnet tasks... Shouldn't it be `[ :vars, :gitlink, :unit_dotnet, :pack_dotnet]`?